app/views/users/new.html.haml

```diff
@@ -11,6 +17,22 @@
           - if @invitation
             = hidden_field_tag :token, @invitation.token
         %fieldset.credit_card
+          .form-group.has-error
+            %span.payment-errors.help-block
+          .form-group
+            %label.control-label.col-sm-2 Credit Card Number
+            .col-sm-6
+              %input.form-control(type="text" data-stripe="number")
+          .form-group
+            %label.control-label.col-sm-2 Security Code
+            .col-sm-6
+              %input.form-control(type="text" data-stripe="cvc")
+          .form-group
+            %label.control-label.col-sm-2 Expiration
+            .col-sm-3
+              = select_month(Date.today, {add_month_numbers: true},  class: 'form-control', data: {stripe: 'exp-month'})
+            .col-sm-2
+              = select_year(Date.today.year, {start_year: Date.today.year, end_year: Date.today.year + 4}, class: 'form-control', data: {stripe: 'exp-year'})
         %fieldset.actions.control-group.col-sm-offset-2
           .controls
-            = f.submit "Sign Up", class: "btn btn-default"
\ No newline at end of file
+            %input(type="submit" value="Sign Up" class="btn btn-default")
```

Comment:
Why did you change it to html? `= f.submit "Sign Up", class: "btn btn-default"` is more preferred.